css-positioning/fixed.css

```diff
@@ -0,0 +1,13 @@
+html {
+  font-size: 22px;
+}
+
+.page-title {
+  font-size: 3rem;
+  text-align: center;
+
+  /* your code here */
+  position: fixed;
+  top: 10px;
+  left: 400px;
```

Comment:
Instead of doing this, you can just set `width: 100%` so that `text-align` can take over centering it for you.